test/integration/warm_up_test.exs

```diff
@@ -0,0 +1,19 @@
+# SPDX-License-Identifier: Apache-2.0
+
+defmodule ChromicPDF.WarmUpTest do
+  use ExUnit.Case, async: false
+
+  describe "ChromicPDF.warm_up/0" do
+    test "fires a one-off chrome task to warmup Chrome's caches" do
+      assert ChromicPDF.warm_up() == {:ok, ""}
+    end
+
+    test "returns stderr when given discard_stderr: false" do
+      # Chrome crashes when given --dump-dom and more than one positional (non --) argument.
+      {:ok, stderr} =
+        ChromicPDF.warm_up(discard_stderr: false, chrome_args: "another_positional_arg")
+
+      assert stderr =~ "Open multiple tabs is only supported when remote debugging is enabled"
```

Comment:
Is this consistent on mac & linux chrome and chromium?

Reply:
tested on mac at least and I know you cover linux, so 👍

Reply:
thank you :purple_heart: